contracts/fee_currencies.go

```diff
@@ -0,0 +1,119 @@
+package contracts
+
+import (
+	"fmt"
+	"math/big"
+
+	"github.com/ethereum/go-ethereum/accounts/abi/bind"
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/ethereum/go-ethereum/contracts/celo/abigen"
+	"github.com/ethereum/go-ethereum/core/vm"
+	"github.com/ethereum/go-ethereum/log"
+)
+
+const (
+	Thousand = 1000
+	Million  = 1000 * 1000
+
+	maxGasForDebitGasFeesTransactions  uint64 = 1 * Million
```

Comment:
Maybe should be just a TODO, but we could limit these numbers more. Technically `Debit + Credit` should be the same gas as the `IntrinsicGasForAlternativeFeeCurrency`, or at least the same order of magnitud.
Technically we could have an erc20 in the future that the transfer consumes more gas (a bug or the erc20 doing something nasty), and with that limit of 1M, we allow them to consume gas that we are not going to charge.
Probably reducing both to 50k is a valid option, but as I said, this could be part of another issue, but we could add it as a TODO

Reply:
I created https://github.com/celo-org/optimism/issues/77 to track this issue. I'll copy your comment over.